config/markets.ts

```diff
@@ -702,6 +702,66 @@ const config: {
       thresholdForStableFunding: decimalToFloat(5, 2), // 5%
       thresholdForDecreaseFunding: decimalToFloat(0), // 0%
     },
+    {
+      tokens: { indexToken: "SHIB", longToken: "WETH", shortToken: "USDC" },
+      virtualTokenIdForIndexToken: hashString("PERP:SHIB/USD"),
+      virtualMarketId: hashString("SPOT:SHIB/USD"),
+
+      ...synthethicMarketConfig, // FIXME or synthethicMarketConfig_IncreasedCapacity ?
+
```

Comment:
missing

      maxLongTokenPoolAmount
      maxShortTokenPoolAmount

      maxLongTokenPoolUsdForDeposit
      maxShortTokenPoolUsdForDeposit

Reply:
asked in chat for these